packages/webapp/src/components/CardWithStatus/TaskCard/TaskCard.jsx

```diff
@@ -67,6 +67,14 @@ export const PureTaskCard = ({
     onClickCompleteOrDueDate?.();
   };
 
+  const isAssigneeInactive = assignee?.status === 'Inactive';
+  let assigneeName = '';
+  if (assignee !== null) {
```

Comment:
@Duncan-Brain I left the null validation just to make sure we don't miss the unassigned state of a task. Let me know if you would llike to see this one differently.

Reply:
looks great!